Monika After Story/game/script-ch30.rpy

```diff
@@ -1170,6 +1170,24 @@ label ch30_end:
 
 # label for things that may reset after a certain amount of time/conditions
 label ch30_reset:
+
+    # setup the docking station to handle the detection
```

Comment:
can you have an initial check so that we only setup the docking station and do all of this if `_mas_pm_has_rpy` is not True?

Also you can just do `not persistent._mas_pm_has_rpy` instead of `persistent._mas_pm_has_rpy is not True`